src/start-app/components/PersonResolver/usePersonResolver.tsx

```diff
@@ -8,49 +8,40 @@ const createPersonClient = (client: IHttpClient) => {
   // TODO - good cache amount? 3min?
   const expire = 3 * 60 * 1000;
   const queryDetails = new Query({
-    expire,
-    queueOperator: 'merge',
-    key: (azureId) => azureId,
-    client: {
-      fn: async (azureId: string) => {
-        const user = await client.json<PersonDetails>(`/persons/${azureId}?api-version=4.0&$expand=positions,manager`);
-
-        try {
-          const image = await client.json<string>(`/persons/${azureId}/photo?api-version=1.0`);
-          user.pictureSrc = image;
-        } catch (error) {
-          // for default image (user.pictureSrc = '/images/profiles/');
-          console.error(error);
-        }
-
-        return user;
+      expire,
+      queueOperator: 'merge',
+      key: (azureId) => azureId,
+      client: {
+          fn: (azureId: string) => {
+              return client.json<PersonDetails>(`/persons/${azureId}?api-version=4.0`);
```

Comment:
Should the new version not include `$expand=positions,manager`?

Reply:
Its not in the frameworks resolver

https://github.com/equinor/fusion-framework/blob/main/packages/cli/src/bin/dev-portal/usePersonResolver.ts

Reply:
Using `<PeropleResolverProvider>` and this will not be used.